docs/command-reference/sorted-sets/zadd.md

```diff
@@ -40,7 +40,7 @@ Note: The **GT**, **LT** and **NX** options are mutually exclusive.
 
 Range of integer scores that can be expressed precisely
 ---
-
+<!-- should it be Redis or Dragonfly on the line below? -->
```

Comment:
@dranikpg Do you know if our representation is the same as Redis'?

Reply:
@benskelker Let's revert this line so that we can merge your (great!) PR. I'll take it offline with @dranikpg and will update as needed, so thanks also for bringing this to my attention!